cesm/driver/esm.F90

```diff
@@ -1324,7 +1328,15 @@ subroutine esm_set_single_column_attributes(compname, gcomp, rc)
           if (status /= nf90_noerr) call shr_sys_abort (subname//' inq_varid frac')
 
           ! Read in domain file for single column
-          allocate(lats(nj))
+          ! Check for unstructured data ni>1 and nj==1
+          if (ni.gt.1 .and. nj == 1) unstructured=.true.
+
+          if (unstructured) then
+             allocate(lats(ni))
+             allocate(pos_lats(ni))
+          else
+             allocate(lats(nj))
```

Comment:
Does this mean lats(j) is the same for the ni points?

Reply:
I'm not sure how this works. It's in the cesm driver and I've never paid attention to it.